### PR TITLE
Fixes strpos for php 7.3

### PR DESCRIPTION
**Pull request by palmiak**
**Ticket**: #1899 

#### Issue
Problem with php 7.3 and strpos - described in ticket.

#### Solution
As stated in PHP docs:
```
If needle is not a string, it is converted to an integer and applied as the ordinal value of a character. This behavior is deprecated as of PHP 7.3.0, and relying on it is highly discouraged. Depending on the intended behavior, the needle should either be explicitly cast to string, or an explicit call to chr() should be performed. 
```

so I added the `chr` function.

#### Impact

None.

#### Usage Changes
None

### CODE REVIEW

**Comment by fluff** [comment]
Adding `chr()` does not seem right at all. Please see my comment in https://github.com/timber/timber/issues/1899#issuecomment-461635642

**Comment by palmiak** [comment]
Just leaving this here to test later:
```
protected static function is_in_theme_dir( $path ) {
    $root = realpath(get_stylesheet_directory());

    if ( false === $root ) {
        return false;
    }

    if ( 0 === strpos($path, (string)$root) ) {
        return true;
    } else {
        return false;
    }
}
```

File: lib/ImageHelper.php [hunk 1]
```diff
@@ -328,7 +328,7 @@ protected static function process_delete_generated_files( $filename, $ext, $dir,
 	 */
 	public static function get_server_location( $url ) {
 		// if we're already an absolute dir, just return
-		if ( 0 === strpos($url, ABSPATH) ) {
+		if ( 0 === strpos($url, chr(ABSPATH)) ) {
```
Comment (fluff) [inline lib/ImageHelper.php]: I don't think this `chr()` should be here either?

Comment (palmiak) [reply lib/ImageHelper.php]: Saw that too - pushed fix already. Thanks.

**Comment by palmiak** [comment]
@jarednova I'm looking at the Travis test results and don't quite understand all the errors. I think that the problem is mostly caused by WP 4.7.9 + PHP 7.3 not the fix itself. 

Do you have any idea how to fix those bugs?

**Comment by jarednova** [comment]
@palmiak according to this post on [WordPress and PHP 7.3](https://make.wordpress.org/core/2018/10/15/wordpress-and-php-7-3/), they aimed for full 7.3 support in the next release, which at that time would have been 5.0. 

Because 4.7.x gets us to +80% in WP version support I'd like to take an honest whack at this one and see what happens if we go up to 4.7.12 for the 4.7 version we test against. Let's see what that does ....

**Comment by palmiak** [comment]
@jarednova 4.7.12 didn't changed anything 😞 We can assume that 4.7.* won't be compatible with 7.3. 

Can we remove just the tests for 4.7 + php 7.3 because they'll always give us error.

**Comment by jarednova** [comment]
Yeah, it appears the WordPress gods have not smiled upon us. I'd like to still test against a "lower" WP version so I just upped that to 4.8.x to see if that makes Travis happy

**Comment by jarednova** [comment]
Ended up going back down to WP 4.7.12 but excluding PHP 7.3 for that combo. All tests are good now!

**Pull request merged**